pkg/cloud/cloud.go

```diff
@@ -314,18 +332,22 @@ func (c *cloud) DeleteFileSystem(ctx context.Context, fileSystemId string) error
 		return err
 	}
 
-	var skipFinalBackup *bool
+	var skipFinalBackupOnDeletion *bool
+	var options []*string
 	for _, tag := range fs.Tags {
-		if *tag.Key == SkipFinalBackupTagKey {
+		if strings.EqualFold(*tag.Key, OptionsOnDeletionTagKey) {
```

Comment:
EqualFold is case-insensitive, so I viewed this as safer in case the customer somehow misconfigured the tag key. But now that I think about it we should be as explicit as we can be here. I think == is probably better? What do you think?

Reply:
I think == is better.